ui-support-service/src/main/java/org/dcsa/uisupport/service/PortCallPartService.java

```diff
@@ -0,0 +1,23 @@
+package org.dcsa.uisupport.service;
+
+import lombok.RequiredArgsConstructor;
+import org.dcsa.uisupport.mapping.PortCallPartMapper;
+import org.dcsa.uisupport.persistence.repository.PortCallPartRepository;
+import org.dcsa.uisupport.transferobjects.PortCallPartTO;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class PortCallPartService {
+  private final PortCallPartRepository portCallPartRepository;
+  private final PortCallPartMapper portCallPartMapper;
+
+  public List<PortCallPartTO> findAll() {
+    return portCallPartRepository.findAll(Sort.by("displayOrder")).stream()
```

Comment:
If this is hardcoded it could be moved to the query method in the repository. That way all the query logic is in the repository class

Reply:
Fixed